crates/orchestrator/src/generators/chain_spec.rs

```diff
@@ -1393,6 +1396,37 @@ mod tests {
         }})
     }
 
+
+    fn chain_spec_with_dev_stakers() -> serde_json::Value {
+        json!({"genesis": {
+            "runtimeGenesis" : {
+                "patch": {
+                    "staking": {
+                        "activeEra": [
+                            0,
+                            0,
+                            0
+                        ],
+                        "canceledPayout": 0,
+                        "devStakers": [
+                            2000,
+                            25000
+                        ],
+                        "forceEra": "NotForcing",
+                        "invulnerables": [],
+                        "maxNominatorCount": null,
+                        "maxValidatorCount": null,
+                        "minNominatorBond": 0,
+                        "minValidatorBond": 0,
+                        "slashRewardFraction": 0,
+                        "stakers": [],
+                        "validatorCount": 500
```

Comment:
Ideally we want to get this from the chainspec.

Reply:
Yes, this is covered in here https://github.com/paritytech/zombienet-sdk/pull/335/files#diff-9cfbdbefcc94bdcfc60d2331da54800823b05adae8f62414ead42659497fa5f2R1449 :)